### PR TITLE
Updated LICENSE.md file with info about mesoshapes

### PR DESCRIPTION
**Pull request by stepps00**
Per https://github.com/whosonfirst/whosonfirst-sources/pull/31, this PR updated the `whosonfirst-data` repo's license file to include information about mesoshapes. 

Since mesoshapes does not have a unique license file, the link on line 25 points to the `meso.json` sources file in the `whosonfirst-sources` repo.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ When we [source](https://github.com/whosonfirst/whosonfirst-sources/) other open
 * `oa` - [OurAirports](http://ourairports.com/) (Public Domain)
 * `qs` - [Quattroshapes](https://github.com/foursquare/quattroshapes/blob/master/) (CC-BY)
 * `zs` - [Zetashapes](http://zetashapes.com/) (CC-BY)
+* `meso` - [Mesoshapes](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/meso.json) (CC-0)
```
Comment (nvkelso) [inline LICENSE.md]: `CC0` (no hyphen, I know right!?)

File: LICENSE.md [hunk 2]
```diff
@@ -233,4 +234,14 @@ flickr api -- you should probably mention on your site if you reuse
...
+
+### Mesoshapes
+
+Properties with the `meso` prefix are from Mesoshapes.
+
+**Examples:** `meso:name_local` for the localized name of a place. `meso:type_loc` for the localized placetype of a feature (simple commune, prefecture, region).
+
+Who's on First doesn't always use the Mesoshapes geometry, so review that in `src:geom`. When we do, some Mesoshapes features source from friendly CC0 or Public Domain licensed projects, and others from a stricter CC-BY project: check that in the `meso:source` property. Features sourced from mesoshapes generally fall in the `locality`, `county`, `macrocounty`, and `region` placetypes.
+
+Mesoshapes are a product of Who's On First under CC0.
```
Comment (nvkelso) [inline LICENSE.md]: Let's remove CC0 here for:

`Mesoshapes are a product of Mapzen.`

File: LICENSE.md [hunk 3]
```diff
@@ -233,4 +234,14 @@ flickr api -- you should probably mention on your site if you reuse
...
+
+### Mesoshapes
+
+Properties with the `meso` prefix are from Mesoshapes.
+
+**Examples:** `meso:name_local` for the localized name of a place. `meso:type_loc` for the localized placetype of a feature (simple commune, prefecture, region).
+
+Who's on First doesn't always use the Mesoshapes geometry, so review that in `src:geom`. When we do, some Mesoshapes features source from friendly CC0 or Public Domain licensed projects, and others from a stricter CC-BY project: check that in the `meso:source` property. Features sourced from mesoshapes generally fall in the `locality`, `county`, `macrocounty`, and `region` placetypes.
+
+Mesoshapes are a product of Who's On First.
```
Comment (stepps00) [inline LICENSE.md]: Taken care of in  	7b22c90

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**